app/src/main/res/layout/fragment_onboarding_icons.xml

```diff
@@ -23,10 +23,11 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="4dp"
+        android:autoSizeTextType="uniform"
         android:layout_marginBottom="28dp"
         android:text="&lt; 11 kW"
         android:textAppearance="@style/TextAppearance.MaterialComponents.Caption"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/welcomeTitle"
```

Comment:
The animation should be closer to the text than to the top of the parent. This can probably be solved using the vertical bias?

Reply:
I wrapped those animations and their text in a `ConstraintLayout `, and I set this block to a suitable position. And now the second page  looks like this:
![Screenshot_1628846392](https://user-images.githubusercontent.com/25502419/129335395-7f83ad5b-6804-45d7-9e6c-1a12cbf4e149.png)
![Screenshot_1628846411](https://user-images.githubusercontent.com/25502419/129335400-87e32b6d-c818-4f2c-a8d8-b63c39043b3f.png)

Reply:
Hm, nested ConstraintLayouts should not be necessary to achieve this. I'll try to come up with something myself.